src/main/java/axoloti/utils/Preferences.java

```diff
@@ -75,6 +78,9 @@
     @Element(required = false)
     String themePath;
 
+    @Element(required = false)
+    String Platform;
```

Comment:
if you felt so strongly, why did you not do this when you added themePath?

also, there are consequences, we are using xml serialisation which currently is case sensitive, so i would also need to change that - just changing things in a cavalier manner is a great way to start introducing bugs.

I do agree though, that stronger adherence to naming conventions is a good idea, but these kind of stylistic changes should be done on dedicated 'style check ins', so they are clear in the version history.

as for your comment about initialising to Platform Axoloti, this is not possible.
this has deliberately not been done because, null for the xml serialisation class means don't save the value, which means I can for now keep the preference 'hidden', if I filled it with PlatformAxoloti it would then be persisted to the prefs file. (unless I set it to null just prior, which is a horrible hack!)